### DISCUSSION TITLE
[Issue Forms] Render checkboxes in collapsible markdown

### DISCUSSION THREAD

**Discussion by xavier2k6** [community/community#5663] [Projects and Issues]
I would like a user to check required "Prerequisite" `checkboxes` before submitting an issue but I would like the "created issue" to show/render the `checkboxes` under a collapsible markdown.

Example:
<details>
<summary>Prerequisite checkboxes</summary>

- [x] I have read the contributing guidelines

- [x] I have confirmed the issue is still present in beta/nightly build

- [ ] some other text

- [x] some other text

</details>

I don't think this is possible currently as docs state markdown isn't rendered.
Can this option/use case be added & if it can any ETA?

**Comment by Bibo-Joshi** [comment]
Possible duplicate of #5250
